lib/mastodon/redis_configuration.rb

```diff
@@ -43,7 +43,7 @@ def setup_config(prefix: nil, defaults: {})
     password = ENV.fetch("#{prefix}PASSWORD", nil)
     host     = ENV.fetch("#{prefix}HOST", defaults[:host])
     port     = ENV.fetch("#{prefix}PORT", defaults[:port])
-    db       = ENV.fetch("#{prefix}DB", defaults[:db])
+    db       = Rails.env.test? ? ENV.fetch('TEST_ENV_NUMBER', defaults[:db]).to_i + 1 : ENV.fetch("#{prefix}DB", defaults[:db])
```

Comment:
Do you need to add `.to_i` to the end?

Reply:
Yes, env vars are strings.

Reply:
sorry. I meant the second fetch.

`Rails.env.test? ? ENV.fetch('TEST_ENV_NUMBER', defaults[:db]).to_i + 1 : ENV.fetch("#{prefix}DB", defaults[:db]).to_i`

Reply:
Presumably, it doesn't really matter because it will be used whether it's a string or not, while we're doing an addition on the former so we need to explicitly convert it to an int.